python/cuml/cuml/tests/experimental/test_filex.py

```diff
@@ -102,7 +102,7 @@ def _build_and_save_xgboost(
     dtrain = xgb.DMatrix(X_train, label=y_train)
 
     # instantiate params
-    params = {"eval_metric": "error", "max_depth": 25}
+    params = {"eval_metric": "error", "max_depth": 25, "device": "cuda"}
```

Comment:
Oh wow! Is it a bug that these tests were training on the host?

Or is it sort of like "it didn't really matter, this was just a way to generate an XGBoost model to test FIL stuff... but we have a GPU available so might as well use it"?

Reply:
It was not a bug per se since what we are testing is FIL indeed, not XGB. And before pytest parallel jobs this don't perform as bad, but once you have 4 or 8 pytest workers, they slow down to a crawl, which we didn't anticipate :)

Reply:
Makes sense! Thanks for explaining that.

Reply:
Not a bug, just an inefficiency. So long as it was a valid model, it was a valid test.